pkg/api/types/system_types.go

```diff
@@ -38,3 +38,22 @@ type SystemEventsOptions struct {
 	// Format the output using the given Go template, e.g, '{{json .}}
 	Format string
 }
+
+// SystemPruneOptions specifies options for `nerdctl system prune`.
+type SystemPruneOptions struct {
+	Stdin  io.Reader
```

Comment:
Is the `Stdin` used in `pkg` side? I think it's only used to confirm if `Force` or not.